src/components/CartPage/CartPage.js

```diff
@@ -95,7 +100,7 @@ const CartPage = (props) => {
             <p>{`$${deliveryCost}`}</p>
             <p>{`$${voucher}`}</p>
             <hr />
-            <p>{`$${countTotal + deliveryCost - voucher}`}</p>
+            <p onChange={getPrice(+`${countTotal + deliveryCost - voucher}`)}>{`$${countTotal + deliveryCost - voucher}`}</p>
```

Comment:
Hmmm, are you sure about template literal?

Reply:
yep, it works)

Reply:
What I see, is inconvinient use of them:
`getPrice(countTotal + deliveryCost - voucher)`

Reply:
Also, its incorrect to put `onChange` on `<p>`element